crates/bevy_color/src/lib.rs

```diff
@@ -151,3 +157,61 @@ where
     Self: Alpha,
 {
 }
+
+macro_rules! impl_componentwise_point {
+    ($ty: ident, [$($element: ident),+]) => {
+        impl std::ops::Add<Self> for $ty {
+            type Output = Self;
+
+            fn add(self, rhs: Self) -> Self::Output {
+                Self::Output {
+                    $($element: self.$element + rhs.$element,)+
+                }
+            }
```

Comment:
Should these clamp by default?

Reply:
I think this would cause a lot of the splines code to work incorrectly with colors.

Reply:
Good point.

Reply:
Bounding indeed breaks many things in splines.